src/new/wallet.js

```diff
@@ -662,7 +662,7 @@ class HathorWallet extends EventEmitter {
       }
       const utxo = utxoDetails.utxos[i];
       inputs.push({
-        tx_id: utxo.tx_id,
+        txId: utxo.tx_id,
```

Comment:
I just saw `tx_id` below. Should we change all to the same pattern? It seems a good opportunity to do it since we will have to update all wallets to use the new signatures.

Reply:
Yeah, I guess you are right. Done.